packages/react-devtools-shared/src/devtools/views/Components/KeyValue.js

```diff
@@ -398,8 +408,12 @@ export default function KeyValue({
           ) : (
             <div className={styles.ExpandCollapseToggleSpacer} />
           )}
-          {renderedName}
-          <div className={styles.AfterName}>:</div>
+          <span
```

Comment:
This span and its inner jsx is repeated three/four times.  Should this be refactored into the RenderedName component? Or a new RenderedNameContainer?